build-system/pr-check.js

```diff
@@ -126,7 +126,9 @@ function isBuildSystemFile(filePath) {
       !isFlagConfig(filePath) &&
       // Exclude visual diff files from build-system since we want it to trigger
       // visual diff tests.
-      !isVisualDiffFile(filePath);
+      !isVisualDiffFile(filePath))
+      // OWNERS.yaml files should trigger build system to run tests
+      || path.basename(filePath) === 'OWNERS.yaml';
```

Comment:
yes, the test already does that. it runs through the repository and creates a temp file to compare with the current CODEOWNERS file